File: src/c/fizzbuzz-4.c
// Author https://stackoverflow.com/users/639428/kenneth
#include <stdio.h>

int i;
for( i = 0; i!=1000000; ++i ) {
    bool bModThree = !(i % 3);
    bool bModFive = !(i % 5);

    if( bModThree || bModFive ) {
        if( bModThree ) {
            printf( "Fizz" );
        }
        if( bModFive ) {
            printf( "Buzz" );
        }
    } else {
        printf( "%d", i );
    }

    printf( "\n" );
}

File: src/js/fizzbuzz-4.js
#!/usr/bin/nodejs
// Author: https://gist.github.com/dstop75
var i, values = [, , 'fizz', , 'buzz', 'fizz', , , 'fizz', 'buzz', , 'fizz', , , 'fizzbuzz'];

for (i = 0; i < 1000000; console.log(values[i++ % 15] || i));
